OpenRA.Game/Network/OrderManager.cs

```diff
@@ -165,14 +165,14 @@ void CheckSync(byte[] packet)
 
 		public bool IsReadyForNextFrame
 		{
-			get { return NetFrameNumber >= 1 && frameData.IsReadyForFrame(NetFrameNumber); }
+			get { return NetFrameNumber >= new FrameID(1) && frameData.IsReadyForFrame(NetFrameNumber); }
```

Comment:
This (and below) can be converted to `NetFrameNumber > FrameID.Zero`.

Reply:
Can I use the `GameStarted` property instead of `NetFrameNumber > FrameID.Zero`?

Reply:
Yes, that should work too.

Reply:
Updated.